resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -561,6 +561,7 @@ public class ApiClient implements AutoCloseable {
         ApiClientConnectorResponse connectorResponse = null;
         try {
             connectorResponse = connector.invoke(connectorRequest);
+            connectorResponse = retryOnFailure(connectorRequest, connectorResponse);
```

Comment:
Wouldn't you want to only enter this method if the request failed with a 429 or 5xx?